src/ui/tokens/tokens_screen.rs

```diff
@@ -2592,12 +2592,7 @@ impl TokensScreen {
                             let mut token_to_remove: Option<u8> = None;
                             for i in 0..self.token_names_input.len() {
                                 ui.label("Token Name (singular):");
-                                ui.horizontal(|ui| {
-                                    ui.text_edit_singleline(&mut self.token_names_input[i].0);
-                                    // Plural name
-                                    ui.label("(plural):");
-                                    ui.text_edit_singleline(&mut self.token_names_input[i].1);
-                                });
+                                ui.text_edit_singleline(&mut self.token_names_input[i].0);
```

Comment:
As per description:

> The whole Searchable token name thing was very confusing. Gave the checkbox a new label and also checked for duplicate keywords and displayed error.

All I see is that the plural was removed, nothing about duplicate keywords.

Reply:
See further down for that part (around comment "// If there are any duplicate keywords, show an error")